lib/ethernet/mac_stm32fxx7.c

```diff
@@ -58,21 +58,26 @@ void eth_set_mac(uint8_t *mac)
 /*---------------------------------------------------------------------------*/
 /** @brief Initialize descriptors
  *
- * @param[in] buf uint8_t* Buffer for the descriptors
- * @param[in] nTx uint32_t Count of Transmit Descriptors
- * @param[in] nRx uint32_t Count of Receive Descriptors
- * @param[in] cTx uint32_t Bytes in each Transmit Descriptor
- * @param[in] cRx uint32_t Bytes in each Receive Descriptor
- * @param[in] isext bool true, if extended descriptors should be used
+ * @param[in] buf uint8_t* Memory area for the descriptors and data buffers
+ * @param[in] nTx uint32_t Count of transmit buffers
+ * @param[in] nRx uint32_t Count of receive buffers
```

Comment:
I think these two are still descriptor counts, not buffer counts right?  Descriptors can have two buffers?